creatorkit/Toolbox.md

```diff
@@ -0,0 +1,135 @@
+![Creator Logo](../creatorlogo.png)
+
+# Toolbox
+
+The toolbox is a reference page for you to use to set up your kit. Follow the steps listed in the project pages and refer back to this page for more detail on individual steps.
+
+## Hardware
+
+* Creator Kit
+* Ubuntu 14.04 64-bit PC/Laptop
+
+## Powering Ci40 + Clicker boards
+
+There are 2 ways to power your Ci40 board.
+
+* Using a power supply
+* Using a microUSB cable connected to the port next to the power supply port
+
+There are also 2 ways to power Clicker boards
+
+* 2 AAA Batteries
+* MicroUSB
+
+## Getting the code
+
+You can get pre-built releases or source code of the Creator Kit projects to work with. The Releases will allow you to set up a working system quickly without having to build any code. The source code will allow you to inspect/extend the projects and build the releases yourself.
+
+[Releases](http://github.com/CreatorKit)
+
+[Source Code](http://github.com/CreatorKit)
```

Comment:
As mentioned this link should be pointing to manifest and that readme can have repo details.